### PR TITLE
fix(orc8r): Fixed reinitialization of cloud health server

### PR DESCRIPTION
**Pull request by christinewang5**
fixed the reinitialization of cloud health server in main.go

### CODE REVIEW

**Review by themarwhal — APPROVED** [review]

File: cwf/gateway/go.mod [hunk 1]
```diff
@@ -44,7 +44,7 @@ require (
 	github.com/stretchr/testify v1.7.0
 	github.com/vishvananda/netlink v1.1.0
 	golang.org/x/crypto v0.0.0-20210921155107-089bfa567519
-	google.golang.org/grpc v1.36.0
```
Comment (nstng) [inline cwf/gateway/go.mod]: Is this change intentional? (not a go expert here)

File: cwf/gateway/go.sum [hunk 2]
```diff
@@ -103,6 +103,7 @@ github.com/dgraph-io/ristretto v0.0.2/go.mod h1:KPxhHT9ZxKefz+PCeOGsrHpl1qZ7i70d
 github.com/dgrijalva/jwt-go v3.2.0+incompatible/go.mod h1:E3ru+11k8xSBh+hMPgOLZmtrrCbhqsmaPHjLKYnJCaQ=
 github.com/dgryski/go-farm v0.0.0-20190423205320-6a90982ecee2/go.mod h1:SqUrOPUnsFjfmXRMNPybcSiG0BgUW2AuFH8PAnS2iTw=
 github.com/dgryski/go-sip13 v0.0.0-20181026042036-e10d5fee7954/go.mod h1:vAd38F8PWV+bWy6jNmig1y/TA+kYO4g3RSRF0IAv0no=
+github.com/dnephin/pflag v1.0.7/go.mod h1:uxE91IoWURlOiTUIA8Mq5ZZkAv3dPUfZNaT80Zm7OQE=
```
Comment (nstng) [inline cwf/gateway/go.sum]: Is this change intentional? (not a go expert here)

File: cwf/gateway/go.mod (hunk 1 above)
Comment (electronjoe) [reply cwf/gateway/go.mod]: Good question Nils. I would think this change is not desired (unless strictly necessary for the PR). @christinewang5 ?

File: cwf/gateway/go.sum (hunk 2 above)
Comment (electronjoe) [reply cwf/gateway/go.sum]: Does our workflow force these `go.mod` and `go.sum` changes, or can they be avoided @christinewang5 ?

Comment (christinewang5) [reply cwf/gateway/go.sum]: Yeah it was auto-added, but with the revert here https://github.com/magma/magma/pull/12500, we probably don't even need this PR anymore, so closing for now.

Comment (nstng) [reply cwf/gateway/go.sum]: I closed #12500 in favor of this task :)

Comment (christinewang5) [reply cwf/gateway/go.sum]: Oh oops, lemme just undo those go.mod and go.sum changes
